packages/libngspice/meta.yaml

```diff
@@ -30,10 +37,24 @@ build:
 
     mkdir release-lib && cd release-lib
     emconfigure ../configure "${configure_args[@]}" CFLAGS="${SIDE_MODULE_CFLAGS}"
+
+    # Build everything except xspice/cmpp first
+    emmake make -j ${PYODIDE_JOBS:-3} -C src/xspice/cmpp LDFLAGS="${SIDE_MODULE_LDFLAGS}"
+
+    # Now replace the wasm cmpp with the native one and touch to prevent rebuild
+    cp ../build-native/src/xspice/cmpp/cmpp src/xspice/cmpp/cmpp
+    touch src/xspice/cmpp/cmpp
+
+    # Continue building the rest
     emmake make -j ${PYODIDE_JOBS:-3} LDFLAGS="${SIDE_MODULE_LDFLAGS}"
     emmake make install
     cp ${WASM_LIBRARY_DIR}/lib/libngspice.so ${DISTDIR}
 
+    # Copy XSPICE code model libraries to ngspice/*.cm
+    # Since shared_library packages extract to /usr/lib/, the final path will be /usr/lib/ngspice/*.cm
```

Comment:
```suggestion
```

This comment is not helpful as the installation path is useful only in runtime not in build time.